docs/src/docs/asciidoc/configuration-model.adoc

```diff
@@ -3,22 +3,231 @@
 :toc: left
 :toclevels: 1
 
-[[oauth2-authorization-server-configurer]]
-== OAuth2AuthorizationServerConfigurer
+[[configure-defaults]]
+== Default configuration
 
-This section is under construction.
+`OAuth2AuthorizationServerConfiguration` is a `@Configuration` that provides the minimal default configuration for an OAuth2 authorization server.
 
-[[oauth2-authorization-server-configuration]]
-== OAuth2AuthorizationServerConfiguration
+`OAuth2AuthorizationServerConfiguration` uses <<configure-custom, `OAuth2AuthorizationServerConfigurer`>> to apply the default configuration and registers a `SecurityFilterChain` `@Bean` composed of all the infrastructure components supporting an OAuth2 authorization server.
 
-This section is under construction.
+[TIP]
+`OAuth2AuthorizationServerConfiguration.applyDefaultSecurity(HttpSecurity)` is a convenience (`static`) utility method that applies the default OAuth2 security configuration to `HttpSecurity`.
+
+The OAuth2 authorization server `SecurityFilterChain` `@Bean` is configured with the following default protocol endpoints:
+
+* xref:protocol-endpoints.adoc#oauth2-authorization-endpoint[OAuth2 Authorization endpoint]
+* xref:protocol-endpoints.adoc#oauth2-token-endpoint[OAuth2 Token endpoint]
+* xref:protocol-endpoints.adoc#oauth2-token-introspection-endpoint[OAuth2 Token Introspection endpoint]
+* xref:protocol-endpoints.adoc#oauth2-token-revocation-endpoint[OAuth2 Token Revocation endpoint]
+* xref:protocol-endpoints.adoc#oauth2-authorization-server-metadata-endpoint[OAuth2 Authorization Server Metadata endpoint]
+* xref:protocol-endpoints.adoc#jwk-set-endpoint[JWK Set endpoint]
+* xref:protocol-endpoints.adoc#oidc-provider-configuration-endpoint[OpenID Connect 1.0 Provider Configuration endpoint]
+* xref:protocol-endpoints.adoc#oidc-user-info-endpoint[OpenID Connect 1.0 UserInfo endpoint]
+
+[NOTE]
+The JWK Set endpoint is configured *only* if a `JWKSource<SecurityContext>` `@Bean` is registered.
+
+[NOTE]
+The xref:protocol-endpoints.adoc#oidc-client-registration-endpoint[OpenID Connect 1.0 Client Registration endpoint] is disabled by default.
+
+The following example shows how to use `OAuth2AuthorizationServerConfiguration` to apply the minimal default configuration:
```

Comment:
Based on earlier feedback, I've been using "The following listing...". Should we standardize on a phrase to use? I think I'd prefer "The following example..." but I'm fine with either.

Reply:
You can mix and match them.

For fussy people like me, there is a distinction to be made. An example is usually the kind of thing that a reader can copy and use (often with modification) in their code, where a listing is a bit of code we offer to show how things work (often under the covers). It really is a fussy point, so leave worrying about that one to me. :)